visma/io/tokenize.py

```diff
@@ -1486,7 +1486,12 @@ def evaluateConstant(constant):
     """
     if isinstance(constant, Function):
         if isNumber(constant.value):
-            return math.pow(constant.value, constant.power)
+            try:
```

Comment:
Here when input is of type {int}^{int}, the constant.value and constant.power are of type list with size one. Using only the following will suffice(no need of try-catch).
```suggestion
            return math.pow(constant.value[0], constant.power[0]) 
```

Reply:
The only reason I used try-catch was to make sure that it doesn't break the functionality elsewhere.
I have corrected it now.
Thanks for clearing it